pkg/images/enricher/metadata.go

```diff
@@ -19,7 +19,8 @@ var (
 	// should be incremented to the new metadataVersion
 	metadataHashToVersion = map[uint64]int{
 		// initial hash of the metadata maps to 0
-		16385665611439321574: 0,
+		// hash changed when mirror* fields were added to storage.DataSource.
+		15313109102692888684: 0,
```

Comment:
That is what I initially did, however a bunch of the enricher tests started failing as a result because by default none set the `version` on the metadata object - so it defaults to 0. 

The intent of this metadata file seems to be to trigger a metadata refresh when needed, and in this case we're adding a datasource field, the existing metadata doesn't need a refresh - so updating the hash only should be OK and reduce the scope of the change.

When sensor local scans are executed, the new fields of the datasource will be set and updated in central organically.